101-bundle-json.md

```diff
@@ -802,7 +802,7 @@ Output specifications are flat (not tree-like), consisting of name/value pairs.
     - `multipleOf`: Output validation requiring that the resulting number be wholly divisible by the number specified. MUST be a number strictly greater than zero. (OPTIONAL)
     - `not`: Output validation requiring that the resulting value NOT match the specified schema. MUST be a JSON schema. (OPTIONAL)
     - `oneOf`: Output validation requiring that the resulting value match ONE of the specified schemas. MUST be a non-empty array of JSON schemas. (OPTIONAL)
-    - `path`: The fully qualified path to a file that will be created (REQUIRED)
+    - `path`: The fully qualified path to a file that will be created (REQUIRED). The path specified MUST be a subpath of /cnab/app/outputs.
```

Comment:
Do we allow distinct outputs to use the same `path` value? I hope not. Probably worth ruling this out with some "MUST NOT" wording.

Reply:
I added that this path must be unique.